### ISSUE TITLE
symlink read as file filetype instead of link

### ISSUE THREAD

**Issue by heranmar** [ranger/ranger#1673]
I don't know if this is a bug or it's just how ranger is, but when I hover over a symlink it doesn't show the appropriate fileype.
![Screenshot from 2019-08-26 11-34-38](https://user-images.githubusercontent.com/52627816/63665736-da4ef280-c7f6-11e9-8a6f-db487be0e76c.png)
ranger interpreted the symlink I'm currently selecting as directory (bottom right drwxr-xr-x), which is supposed to be a link filetype (lrwxr-xr-x).

I just recently using ranger, so I don't know if this is how it supposed to be but it would be neat to show an appropriate file information. I also think it would be a good feature to add where ranger can show the symlink path.

**Comment by toonn** [comment]
The color and the `->` indicate it's a symlink. Why would you not want symlinks to be treated like what they point to? That's usually why you'd use a symlink. Symlinks that don't point to a valid target are colored differently and their target is shown in the statusbar.

I wouldn't consider a setting to treat symlinks differently as an impossibility though. Do you have a good motivating usecase?

**Comment by Vifon** [comment]
While I mostly agree with all you said, @toonn, I agree with @heranmar that the permissions should start with `l` and not `d` in this case. Especially since we already do that for both broken symlinks and for symlinks to files.

**Comment by heranmar** [comment]
For example, If I want to see where the symlink is pointing to, without having to navigate inside the symlink to perform a quick scan of all the symlink inside a directory. Having to navigate inside each link just to know the precise file path is pretty tedious.

In normal scenario, I can use command `file [link filename]` in order to see where exactly the link is pointing to. I can manage with doing it manual, but wouldn't it be neat to see where the link is heading to without exiting ranger first then use command `file` manually one by one? or maybe make a certain command in ranger to show where the link is heading to.

**Comment by toonn** [comment]
@Vifon, I disagree. The relevant permissions are those of the target. Permissions on the link are pretty much irrelevant. Showing an `l` with the permissions of the target is preference at most imo, not a bug. So I'd remove the bug label.

I'd much rather see a setting similar to what most programs have `set follow_links true`'d be the default and if you turn it off you get your `l` and the preview's the `file(1)` output.

**Comment by heranmar** [comment]
You had a point. But will the option to show where the link is heading to be implemented? I feel like configuration like `set follow_links true` are more of an each individual taste and should be changeable should the user wish to. Is there such setting that I can change?

**Comment by toonn** [comment]
@heranmar, there's not, that's why I'm suggesting implementing this.

**Comment by Vifon** [comment]
@heranmar, in a wider terminal, the target is shown in the status line. It gets shown for you because of the limited horizontal space.
